app/component/itinerary/NaviCardContainer.js

```diff
@@ -98,6 +85,7 @@ function NaviTop(
       if (legChanged) {
         focusToLeg?.(newLeg);
```

Comment:
if ( newLeg) ...

Reply:
I'm confused as why? 

 newLeg is defined like this:   ```const newLeg = realTimeLegs.find(leg => {
      return legTime(leg.start) <= time && time <= legTime(leg.end);
    });``` in useEffect.

    So basically every 10 seconds we get the same leg until we eventually get the new leg. Thats why I introduced the `legChanged` param to handle the situation where the leg is actually changed.
    
  I guess we might want to set focus always when we have new leg? But In my mind currentLeg should be set when there is a change in legs. (Also the card should not be collapsed every 10 seconds ofc.)

Reply:
Done.